marklogic-data-hub-central/ui/src/components/detail-page-non-entity/detail-page-non-entity.test.tsx

```diff
@@ -1,12 +1,62 @@
 import React from "react";
-import {render} from "@testing-library/react";
+import {render, fireEvent, waitForElement} from "@testing-library/react";
 import {BrowserRouter as Router} from "react-router-dom";
 import DetailPageNonEntity from "./detail-page-non-entity";
 import testData from "../../assets/mock-data/explore/Non-entity-document-payload";
 import userEvent from "@testing-library/user-event";
+import axiosMock from "axios";
+
+jest.mock("axios");
 
 describe("Detail page for non-entity view component", () => {
 
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const response =
```

Comment:
It would be better if we place the custom response in a separate file, may be under "assets/mock-data/explore/" in a response file specifically for Non-Entity documents.